sstables/sstables.cc

```diff
@@ -3155,24 +3155,22 @@ delete_atomically(std::vector<shared_sstable> ssts) {
 // since this is an indication we crashed in the middle of delete_atomically
 future<> replay_pending_delete_log(sstring pending_delete_log) {
     sstlog.debug("Reading pending_deletes log file {}", pending_delete_log);
-    return seastar::async([pending_delete_log = std::move(pending_delete_log)] {
         sstring pending_delete_dir = parent_path(pending_delete_log);
```

Comment:
distributed_loader::handle_sstables_pending_delete doesn't limit concurrency of log replay, so it's good that you're removing the thread usage, where each consumes 128k for its stack. Unlikely to have any impact in practice though.